advanced/src/hashTable.js

```diff
@@ -0,0 +1,227 @@
+function Node(key, value) {
+  this.key = key
+  this.value = value
+  this.next = null
+}
+
+
+const LinkedList = class LinkedList {
+  constructor() {
+    this.length = 0
+    this.head = null
+  }
+
+  insert(key, value) {
+    let node = new Node(key, value)
+    let currentNode = this.head
+
+    if (!currentNode) {
+      this.head = node
+      this.length++
+      return node
+    }
+    while (currentNode.next) {
+      currentNode = currentNode.next
+    }
+    currentNode.next = node
+    this.length += 1
+    return node
+  }
+
+//regular find
+  find(key) {
+    var currentNode = this.head
+    if (!currentNode) {
+      return false
+    }
+    while (currentNode) {
+      if (currentNode.key === key){
+        return currentNode
+      }
+    } return false
```

Comment:
Drop this down a line; it's a little confusing to have the `return` statement on the same line as the closing brace for the while block.